doc/getting-started.md

```diff
@@ -138,7 +151,7 @@ dp train input.json
 
 If enbled in json/yaml input file DeePMD-kit will create log files which can be
 used to analyze training procedure with Tensorboard. For a short tutorial
-please read this [document](doc/tensorboard.md).
+please read this [document](tensorboard.html).
```

Comment:
Is it a valid link? I do not see tensorboard.html in our source

Reply:
This link is valid. All the html files are compiled and under directory _build.

Reply:
I changed it back anyway.